Assets/Scripts/Persona/api/APIClient.cs

```diff
@@ -113,5 +111,62 @@ public async Task<string> MakePostRequestAsync(string url, string postData, Dict
                 }
             }
         }
+
+        public async Task<Texture2D> DownloadImageFromUrl(string mediaUrl)
```

Comment:
This function looks more like a utility function and we can probably move it to a utils file